heclib/heclib_c/src/headers/hecdss7.h

```diff
@@ -41,7 +41,7 @@ int zclose(long long *ifltab);
 int zgetFileVersion(const char *dssFilename);
 int zgetVersion(long long *ifltab);
 int zgetFullVersion(long long *ifltab);  //  For example DSS Version "7-BG" = 70207
-int zopen(long long *ifltab, const char *dssFilename);
+int hec_dss_zopen(long long *ifltab, const char *dssFilename);
```

Comment:
Didn't Karl already make this change? Instead of making it again merge or rebase master onto this branch.

Reply:
Don't know what to say about this one. I executed `git pull origin master` before I created my local branch, so I was surprised to find the conflict when I tried to compile on my Macbook.

Reply:
weird, and the PR isn't flagging a conflict so, heh.